### DISCUSSION TITLE
Disable accept-suggestion on `Enter` press

### DISCUSSION THREAD

**Discussion by spaceunifyfifty** [zed-industries/zed#15845] [Feature Requests]
When I type `do` in scala and then press enter to go onto a newline, Zed insteads accepts the `double` suggestion popup so I keep accidentally having `for n <- seq double`. I would like to turn this off, and have accept-suggestions only triggered on `Tab` press. I can't seem to find the setting for this. I assume there's got to be one already, right?

**Comment by sleeping-at-night** [comment]
Could it be this? 

```json
[
  {
    "context": "Editor && showing_completions",
    "bindings": {
      "enter": "editor::ConfirmCompletion",
      "tab": "editor::ConfirmCompletion"
    }
  }
]

```

**Reply by sleeping-at-night** [reply]
Change the value of `"enter"` from `"editor::ConfirmCompletion"` to `null` in the setting above seem to working for me.

**Reply by spaceunifyfifty** [reply]
> Change the value of `"enter"` from `"editor::ConfirmCompletion"` to `null` in the setting above seem to working for me.

`null` actually results in `newline` being disabled too, forcing me to press `esc` to close the popup which was the problem I'm trying to solve anyway. This is the setting I found that results in my desired  behavior
```json
[
    {
        "context": "Editor && showing_completions",
        "bindings": {
            "enter": "editor::Newline",
            "tab": "editor::ConfirmCompletion"
        }
    }
]
```

I hadn't thought to check `keymap`, only the `settings`. Was obvious to find there. Thanks!
